app/models/participant_profile.rb

```diff
@@ -35,10 +35,21 @@ class ParticipantProfile < ApplicationRecord
     withdrawn: "withdrawn",
   }, _prefix: "training_status"
 
+  attribute :reason_for_new_cohort, :string
+  enum reason_for_new_cohort: {
+    dqt_mismatch_induction_start_date: "dqt_mismatch_induction_start_date",
+    lead_provider_change: "lead_provider_change",
+    payments_frozen_at_previous_cohort: "payments_frozen_at_previous_cohort",
```

Comment:
I was only aware of this reason; what do the others mean?

Reply:
- `dqt_mismatch_induction_start_date` means that our induction start date mismatch that in DQT and therefore we moved the participant to the cohort calculated from DQT's one.

- `lead_provider_change` means basically the api throught the `change_schedule` endpoint 

But currently these won't be of any use

Reply:
Im not sure about this, the payments frozen is quite different than the others, the others mean they changed cohort, the payments frozen means they span across both, not sure we should be grouping them the same?